Lesson 11/HW11.cs

```diff
@@ -0,0 +1,30 @@
+namespace HW456.Lesson_11;
+
+/// <summary>
+/// Написать метод-расширение для массива целых чисел, который принимает как параметр строку-разделитель,
+/// а возвращает строку, в которая содержатся все элементы массива перечисленные через указанный разделитель.
+/// </summary>
+
+public static class HW11
+{
+    public static void Task()
+    {
+        Console.WriteLine("Input splitter: ");
+        string splitter = Console.ReadLine();
+
+        int[] array = Lesson11.GenerateArray();
+        
+        Console.WriteLine(splitter.ArrayWithSplitter(array));
+    }
+    public static string ArrayWithSplitter(this string splitter, int[] arr)

```

Comment:
По условию задания `Написать метод-расширение **для массива целых чисел**, который принимает как **параметр строку-разделитель**` метод написан немного неверно.

Он должен быть для массива целых чисел, то есть:
```suggestion
    public static string ArrayWithSplitter(this int[] arr, string splitter)

```

И минор: устоявшееся слово для разделителя - `separator`. Для примера можно посмотреть `string.Join()`